webapp/index.html

```diff
@@ -8,7 +8,7 @@
 
 	<script id="sap-ui-bootstrap" type="text/javascript"
 		src="resources/sap-ui-core.js"
-		data-sap-ui-theme="sap_bluecrystal"
+		data-sap-ui-theme="sap_belize"
```

Comment:
Better now and yes, it [is available](https://github.com/openui5/packaged-themelib_sap_belize). Thanks 👍